Assets/MixedRealityToolkit-Examples/MotionControllers-GrabMechanics/Scripts/BaseGrabber.cs

```diff
@@ -60,7 +60,7 @@ public Transform GrabHandle
         {
             get
             {
-                return grabAttachSpot != null ? grabAttachSpot : transform;
+                return grabAttachSpot ?? transform;
```

Comment:
Please revert. Null Coalescence is not supported in Unity.

https://github.com/JetBrains/resharper-unity/issues/323

Reply:
Yep, I also fell for that mistake and caused a few errors with it. It's tempting to let ReSharper change it for you and I'm working on modifying the unity plugin to change that.

Reply:
Ok, I'll revert this one, even though we created a separate issue to address this accros the project :D

Reply:
Done